school/tests.py

```diff
@@ -1,3 +1,144 @@
+from school.models import Parent
 from django.test import TestCase
+from rest_framework.reverse import reverse
 
-# Create your tests here.
+class ParentAPITest(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.list_url = reverse('parent list')
+        # cls.paginated_url = reverse('presets:industry-type-option-paginated')
+        cls.model = Parent
+        cls.queryset = cls.model.objects.all()
+        # cls.factory = preset_factory.IndustryTypeOptionFactory
+
+    def setUp(self):
+        super().setUp()
+        self.parent_instance = Parent.objects.create(
+          id = 3,  first_name = "Jane", last_name = "Smith", phone = "+41524204242"
+        )
+
+    def test_list(self):
+        response = self.client.get(self.list_url)
+        self.assertEqual(response.status_code, 200)
+        expected_response = [
+            {
+                "id" : 3,
+                "first_name": "Jane",
+                "last_name": "Smith",
+                "phone" : "+41524204242"
+            }
+        ]
+        self.assertEqual(response.json(), expected_response)
+
+#    def test_paginated(self):
+#         response = self.client.get(
+#             self.paginated_url
+#         )
+#         self.assertEqual(response.status_code, 200)
+#         expected_response = {
+#             "count": 1,
+#             "previous": None,
+#             "next": None,
+#             "results": [
+#                 {
+#                     "label": "ECOMMERCE",
+#                     "value": "Ecommerce"
+#                 }
+#             ]
+#         }
+#         self.assertEqual(response.json(), expected_response)
+
+    # def test_detail(self):
+    #     response = self.client.get(
+    #         f"{self.list_url}{self.default_instance.uuid}/"
+    #     )
+    #     self.assertEqual(response.status_code, 404)
+
+    def test_create(self):
+        data = {
+            "id" : 3,
```

Comment:
remove "id": 3

Reply:
could not remove "id" : 3 in test_create and as well in test_list "id" : 2. If 'ID' removed, shows 'Assertion Error' : List diff.... . 
May be because I added Id in serializers.py.